src/core_plugins/testbed/public/testbed.html

```diff
@@ -5,6 +5,14 @@
     you need to demonstate your functionality. Nothing need be preserved. -->
     <!-- CONTENT START -->
 
+    <div class="kuiControlledTable kuiVerticalRhythm">
+      <no-items></no-items>
+    </div>
+
+    <div class="kuiControlledTable kuiVerticalRhythm">
+      <no-items items-name="things"></no-items>
```

Comment:
I have the same comments here as I did in https://github.com/elastic/kibana/pull/11778#discussion_r116657653